content/1_General/Contributing.mdx

```diff
@@ -195,4 +194,54 @@ Additional Notes:
   print(a + b)
   ```
 
-  It doesn't matter which one you use (`a+b` or `a + b`), but pick one and stick with it!
\ No newline at end of file
+  It doesn't matter which one you use (`a+b` or `a + b`), but pick one and stick with it!
+
+<LanguageSection>
+
+<CPPSection>
+
+#### C++ Template
+
+For Bronze through Gold, please only use the macros listed in the template below (see [Github #806](https://github.com/cpinitiative/usaco-guide/issues/806)):
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+using ll = long long;
+using vi = vector<int>;
+using pi = pair<int,int>;
+
+#define sz(x) (int)(x).size()
+#define pb push_back
+#define all(x) begin(x), end(x)
+#define f first
+#define s second
+
+int main() {
+    cin.tie(0)->sync_with_stdio(0); 
+    // freopen("name.in", "r", stdin); 
+    // freopen("name.out", "w", stdout);
+
+}
+```
+
+You don't have to use the template, but avoid using additional macros beyond the ones listed above.
```

Comment:
@bqi343 is this fine?

Reply:
https://github.com/cpinitiative/usaco-guide/issues/806#issuecomment-782051462 maybe?